_posts/2017-08-21-police-malpractice.md

```diff
@@ -0,0 +1,50 @@
+
+One path forward is to require each officer to hold liability insurance, rather than being covered by the department or city. The officers who killed George Floyd were [repeatedly reprimanded](https://www.mprnews.org/story/2015/11/19/the-officers-in-the-jamar-clark-shooting) for excessive force. The officers who killed Jamal Clark had a [history of violence](https://www.mprnews.org/story/2015/11/19/the-officers-in-the-jamar-clark-shooting) as well. Each of these men was a multimillion-dollar settlement waiting to happen, which any insurance underwriter could have seen. Prohibitive insurance costs should have made them unemployable as police officers long before they had a chance to kill, regardless of support from the department or union.
+
+<!-- the cost of insurance could also be shared by superior officers, creating an incentive not to hire high-risk officers. but that's a next-order effect, and explaining might be more of a distraction than it's worth -->
+
+Addressing police misconduct through civil suits and liability insurance may also help shift public opinion. Under [today's laws](https://en.wikipedia.org/wiki/Qualified_immunity), it's [hard to convict the police](https://www.politico.com/news/2020/06/01/minnesota-attorney-general-hard-to-convict-police-293920). They're [rarely even indicted](https://fivethirtyeight.com/datalab/ferguson-michael-brown-indictment-darren-wilson/). But criminal cases attract far more attention than civil suits (see chart below). A [consistent majority](http://www.gallup.com/poll/213869/confidence-police-back-historical-average.aspx) of Americans report a great deal of confidence in the police. To advance reforms at the ballot box, that confidence needs to be shaken. Putting million-dollar settlements front and center might do the trick. "[Officer who shot Philando Castile found not guilty on all counts](http://www.cnn.com/2017/06/16/us/philando-castile-trial-verdict/index.html)" won't.
+
+<!-- this wording still isn't quite right. would like to note that this is the *climactic* headline. the one thing that everyone reads.  -->
+
+![In terms of search traffic, the climax of the Philando Castile case was Officer Yanez's acquittal, not the $3M wrongful death settlement.](/assets/images/wide/malpractice-castile-trends-16x9.png)
+*In terms of [search traffic](https://g.co/trends/9eJeo), the climax of the Philando Castile case was Officer Yanez's acquittal. The family's $3M wrongful death settlement with the city received comparably little attention.*
+
+In the healthcare industry, malpractice insurance puts a price tag on risk. High-risk doctors are more expensive to employ, and hospitals save money by training staff to avoid mistakes. Requiring police officers to hold liability insurance would work similarly, saving taxpayer dollars and decreasing the use of excessive force.
```

Comment:
```suggestion
In the healthcare industry, malpractice insurance puts a price tag on risk. High-risk doctors are more expensive to employ, and hospitals save money by training staff to avoid mistakes. This increases the public's trust in doctors. Requiring police officers to hold liability insurance would have similar effects; it would save taxpayer dollars and decrease the use of excessive force.
```

Reply:
Updated to call out public trust earlier. Not sure it's central enough to go in the conclusion